www/ui-overlays.html

```diff
@@ -816,3 +816,55 @@ <h3>Overlays and rotation</h3>
 });
 </pre>
 </div>
+
+
+<div class="description">
+    <h3>Overlays and events</h3>
+    <p>
+        Events fired on elements hidden inside elements that are part
+        of the viewer are managed by <code>MouseTracker</code> system.
+        Trying to use vanilla JS or other utilities such as jQuery will
+        fail.
+    </p>
+</div>
+<div class="demoarea">
+<script type="text/javascript">
+    rotationViewer.addHandler('open', () => {
+        new OpenSeadragon.MouseTracker({
+            element: document.getElementById('overlay-rotation-exact'),
+            pressHandler: e => alert('Element clicked! ' + e.originalEvent.target.id),
+            contextMenuHandler: e => alert('Element clicked! ' + e.originalEvent.target.id),
+        });
+        new OpenSeadragon.MouseTracker({
+            element: document.getElementById('right-arrow-overlay-no-rotate'),
+            pressHandler: e => alert('Element clicked! ' + e.originalEvent.target.id),
+            contextMenuHandler: e => alert('Element clicked! ' + e.originalEvent.target.id),
+        });
+        new OpenSeadragon.MouseTracker({
+            element: document.getElementById('overlay-rotation-bounding-box'),
+            pressHandler: e => alert('Element clicked! ' + e.originalEvent.target.id),
+            contextMenuHandler: e => alert('Element clicked! ' + e.originalEvent.target.id),
+        });
+    });
+</script>
+Try to click on the overlay in the above example.
```

Comment:
Yeah, it is a long page; I can see the appeal of not adding another viewer. It is a little confusing, though. Maybe a tweak to the wording can help. Here's a proposed version: 

`Click on an overlay in the "Rotation" example above.`